build.sbt

```diff
@@ -103,7 +104,6 @@ lazy val http4s = project
       "org.http4s"    %% "http4s-circe"        % "0.21.0-M5",
       "org.http4s"    %% "http4s-blaze-server" % "0.21.0-M5",
       "io.circe"      %% "circe-parser"        % "0.12.3",
-      "io.circe"      %% "circe-derivation"    % "0.12.0-M7",
```

Comment:
If we reduce the dependency to circe-core as I suggested in the other comment, this is good. But otherwise, I think we should keep that one so that people who use caliban-http4s don't need to add the dependency to circe-derivation themselves.